geode-assembly/src/distributedTest/java/org/apache/geode/management/internal/rest/ClientClusterManagementSSLTest.java

```diff
@@ -124,13 +117,11 @@ public void createRegion_NoSsl() throws Exception {
     int httpPort = locator.getHttpPort();
 
     client.invoke(() -> {
-      ClusterManagementServiceConfig config = JavaClientClusterManagementServiceConfig.builder()
-          .setHost("localhost")
-          .setPort(httpPort)
-          .setUsername("dataManage")
-          .setPassword("dataManage")
+      ClusterManagementService cmsClient = ClusterManagementServiceBuilder.buildWithUrl()
+          .setCredentials("dataManage", "dataManage")
+          .setUrl("localhost", httpPort)
+          .setSslContext(null)
```

Comment:
is `setSslContext(null)` necessary? Wouldn't it already be the default?

Reply:
this is necessary when we need to run tests multiple times in StressNewTest